flink-table/flink-table-api-java/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,10 @@
+flink-table
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details.
+
+- com.ibm.icu:icu4j:65.1
```

Comment:
This is only needed when the module bundled such dependency. I don't think `flink-table-api-java` would bundle this, maybe `flink-table-planner` and `flink-table-planner-blink` should do such thing.

Reply:
It's also my concern. 
one approach is we does not support this feature in `TableResult.print`. consider that the most uses case of `TableResult.print` is used in IDE for debug and demo. In IDEA, even we use `icu4j` to get word width, the result also looks irregular when the content contain CJK.
see the comment in [CliTableauResultViewTest#testStreamingResult](https://github.com/apache/flink/blob/master/flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliTableauResultViewTest.java#L290)

Reply:
Could we have this dependency in `flink-table-api-java` but only bundle it in planner uber jar?

Reply:
do you mean we set its scope as `provided`, and package this jar into uber jar ?